doc/user_guide/configuration/index.rst

```diff
@@ -0,0 +1,15 @@
+=============
+Configuration
+=============
+
+Pylint is highly configurable. There's a lot of options to follow the need of
+every projects and a lot of checks to activate if they suit your style.
+
+.. toctree::
+   :maxdepth: 2
+   :titlesonly:
+
+   all-options
+   features
```

Comment:
All options and features are really close in content, should we only keep features and rename it pylint's options ?

Reply:
Is there anything in `features` that isn't covered by `messages` and `all-options`?

Reply:
Ha I suppose features is the one we should suppress, but I liked its appearance better than all-options.